core/src/main/java/org/elasticsearch/index/query/SpanMultiTermQueryBuilder.java

```diff
@@ -42,6 +60,42 @@ protected void doXContent(XContentBuilder builder, Params params)
     }
 
     @Override
+    protected Query doToQuery(QueryParseContext parseContext) throws IOException {
+        Query subQuery = multiTermQueryBuilder.toQuery(parseContext);
+        if (subQuery instanceof DateFieldMapper.DateFieldType.LateParsingQuery) {
+            subQuery = ((DateFieldMapper.DateFieldType.LateParsingQuery) subQuery).rewrite(null);
```

Comment:
add a simple comment, just so we know why we do this and why we live with that although it sucks? :)